### ISSUE TITLE
Naming of training images

### ISSUE THREAD

**Issue by mertmerci** [Banconxuan/RTM3D#17]
Hello,

How should the images under `./kitti_format/data/kitti/image` be named? Left images are named like `000000.png` ... `007480.png`. Are the right ones should `007481.png` ... `014961.png`? 

Thank you for your consideration.

**Comment by walzimmer** [comment]
Yes you are right. In total you should have 14962 images in that folder.
